api/optimeet/optimeet/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for optimeet project.
+
+Generated by 'django-admin startproject' using Django 4.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-h_*173ykib^8x0m6@elbqelzysfden)m#m1%7==btv%9i@9qr('
```

Comment:
We shouldn't leave secret keys in Git repositories for security reasons. Since this one will remain in the Git history even if you remove it with another commit, you should generate a new one. Create a new Google Doc where we'll keep all future secret keys, and put the new key there. Share the doc in Discord plz! In order to get the key in this file, we'll want to pass it in as an environment variable. You can see examples of how to do all this at this link: https://www.makeuseof.com/django-secret-key-generate-new/

Reply:
Done